v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/ExtractionPipeline.scala

```diff
@@ -0,0 +1,245 @@
+          V2FExtractions.convertJsonFieldsValueType(
+            VariantEffectRegulatoryFeatureConsequences.tableName,
+            currentfieldsToConvertToJsonArray._2,
+            V2FExtractions.jsonStringToJsonArray(
+              delimeter = currentfieldsToConvertToJsonArray._1
+            )
+          )(currentTransformedJsonAndFilePaths)
+      }
+
+    // convert given fields to json double
+    val variantEffectTranscriptConsequencesTransformedDoubles =
+      V2FExtractions.convertJsonFieldsValueType(
+        VariantEffectTranscriptConsequences.tableName,
+        VariantEffectTranscriptConsequences.fieldsToConvertToJsonDouble,
+        V2FExtractions.jsonStringToJsonDouble
+      )(variantEffectTranscriptConsequencesJsonAndFilePaths)
+
+    // convert given fields to json int
+    val variantEffectTranscriptConsequencesTransformedInts =
+      V2FExtractions.convertJsonFieldsValueType(
+        VariantEffectTranscriptConsequences.tableName,
+        VariantEffectTranscriptConsequences.fieldsToConvertToJsonInt,
+        V2FExtractions.jsonStringToJsonInt
+      )(variantEffectTranscriptConsequencesTransformedDoubles)
+
+    // then convert given fields to json booleans
+    val variantEffectTranscriptConsequencesTransformedBools =
+      V2FExtractions.convertJsonFieldsValueType(
+        VariantEffectTranscriptConsequences.tableName,
+        VariantEffectTranscriptConsequences.fieldsToConvertToJsonBoolean,
+        V2FExtractions.jsonStringToJsonBoolean
+      )(variantEffectTranscriptConsequencesTransformedInts)
+
+    // then convert given fields to json arrays
+    val variantEffectTranscriptConsequencesTransformedArrays =
+      VariantEffectTranscriptConsequences.fieldsToConvertToJsonArray.foldLeft(
+        variantEffectTranscriptConsequencesTransformedBools
+      ) {
+        case (currentTransformedJsonAndFilePaths, currentfieldsToConvertToJsonArray) =>
+          V2FExtractions.convertJsonFieldsValueType(
+            VariantEffectTranscriptConsequences.tableName,
+            currentfieldsToConvertToJsonArray._2,
+            V2FExtractions.jsonStringToJsonArray(
+              delimeter = currentfieldsToConvertToJsonArray._1
+            )
+          )(currentTransformedJsonAndFilePaths)
+      }
+
+    // then convert given fields of the array from json strings to json double
+    val variantEffectTranscriptConsequencesTransformed =
+      V2FExtractions.convertJsonFieldsValueType(
+        VariantEffectTranscriptConsequences.tableName,
+        VariantEffectTranscriptConsequences.fieldsToConvertFromJsonArrayStringToDouble,
+        V2FExtractions.convertJsonArrayStringToDouble
+      )(variantEffectTranscriptConsequencesTransformedArrays)
+
+    // Write all the converted/parsed json objects to disk.
+    frequencyAnalysisTransformed.map {
+      case (jsonObj, _) =>
+        jsonObj
+    }.saveAsJsonFile(
+      s"${parsedArgs.outputDir}/frequencyAnalysis"
```

Comment:
I would make the output paths consistently use kebab-case:
```suggestion
      s"${parsedArgs.outputDir}/frequency-analysis"
```